sweetpea/__init__.py

```diff
@@ -76,6 +77,30 @@ def print_experiments(block: Block, experiments: List[dict]):
         print(reduce(lambda a, b: a + format_str.format(*b), transposed, ''))
 
 
+"""
+Export the generated experiments to csv files. Each experiment will be exported to a separate csv file. 
+"""
+def experiment_to_csv(experiments: List[dict], file_prefix = "experiment"):
```

Comment:
We might want print an error in case no trials sequences are found like in print_experiments.
Other than that this seems great.

Reply:
In case you are wondering about the failing checks, they have been fixed in another branch I'll merge that too.

Reply:
Great, thank you! I will add the "print error" piece.